### PR TITLE
Remove unecessary join to Territory table from progress layer queries

### PR DESCRIPTION
**Pull request by maurizi**
This was part of #1313, I am pulling it out because it is a smaller simpler change with less risk involved

### CODE REVIEW

File: src/tiler/sql/user_progress.sql [hunk 1]
```diff
@@ -2,14 +2,12 @@
 -- the latest survey for a blockface
 (SELECT
   DISTINCT ON (block.id)
-  block.geom, block.id, turf.group_id,
+  block.geom, block.id
```
Comment (jwalgran) [inline src/tiler/sql/user_progress.sql]: This crashes the tiler because a comma is missing before the ``CASE``

**Comment by jwalgran** [comment]
The progress pages function normally. The more complex popup request is still only 10s of milliseconds. Running JMeter against my dev VM showed a couple percent decrease in average request time.

File: src/tiler/sql/user_progress.sql (hunk 1 above)
Comment (maurizi) [inline src/tiler/sql/user_progress.sql]: Fixed

**Comment by jwalgran** [comment]
+1

**Pull request merged**